release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1957,7 +1957,9 @@ There is no workaround for this issue. (link:https://issues.redhat.com/browse/OC
 
 * {product-title} clusters that are installed on {aws-short} in the Mexico Central region (`mx-central-1`) cannot be destroyed. (link:https://issues.redhat.com/browse/OCPBUGS-56020[*OCPBUGS-56020*])
 
-* When installing a cluster on {azure-short}, if you set `compute.platform.azure.identity.type`, `controlplane.platform.azure.identity.type`, or `platform.azure.defaultMachinePlatform.identity.type` to "None", your cluster is unable to pull images from the Azure Container Registry. You can avoid this issue by providing a user-assigned identity, or by leaving the identity field blank, which causes the installation program to generate a user-assigned identity. (link:https://issues.redhat.com/browse/OCPBUGS-56008[*https://issues.redhat.com/browse/OCPBUGS-56008*])
+* When installing a cluster on {azure-short}, if you set `compute.platform.azure.identity.type`, `controlplane.platform.azure.identity.type`, or `platform.azure.defaultMachinePlatform.identity.type` to "None", your cluster is unable to pull images from the Azure Container Registry. You can avoid this issue by providing a user-assigned identity, or by leaving the identity field blank, which causes the installation program to generate a user-assigned identity. (link:https://issues.redhat.com/browse/OCPBUGS-56008[*OCPBUGS-56008*])
+
+* When installing a cluster on {aws-short}, if you use the `openshift-install` binary to create the `install-config.yaml` file or create a cluster, and you did not provide {aws-short} credentials in the installation directory or in the `AWS_PROFILE` environment variable, the installation program will fail. (link:https://issues.redhat.com/browse/OCPBUGS-56658[*OCPBUGS-56658*])
```

Comment:
> you did not provide {aws-short} credentials in the installation directory or in the `AWS_PROFILE` environment variable

This is not accurate I think, it's more like _did not config AWS credentials correctly_, which includes:
1. The default AWS  credential file `~/.aws/credential` does not exist.
2. The default AWS  credential file `~/.aws/credential` exists, but the corresponding profile (the value of `AWS_PROFILE`) does not present in credential file
3. _other cases_

Reply:
Thanks Yunfei, I've updated the wording